static/js/components/dashboard/courses/Progress.js

```diff
@@ -0,0 +1,39 @@
+// @flow
+/* global SETTINGS: false */
+import React from 'react';
+import urljoin from 'url-join';
+import R from 'ramda';
+
+import { formatGrade } from '../util';
+import type { CourseRun } from '../../../flow/programTypes';
+import { EDX_LINK_BASE } from '../../../constants';
+
+const progressHref = courseId => urljoin(EDX_LINK_BASE, courseId, "progress");
+
+type ProgressType = {
+  courseRun: CourseRun,
+  className: string,
+};
+
+// flow disagrees with me here
```

Comment:
oh, basically flow thought that it was possible for `courseRun.current_grade` to be `undefined`, so I had to add a coercion to `any` here. we can delete the comment if it isn't necessary :)